### ISSUE TITLE
Thai doesn't read its control letters

### ISSUE THREAD

**Issue by boomwooq** [carrois/Kernkraft#30]
I think, it's no longer read Thai script
![CleanShot 2024-05-09 at 20 24 36@2x](https://github.com/bBoxType/Kernkraft/assets/15804957/5f7b564f-945b-428a-9001-026165d868da)

**Comment by Mark2Mark** [comment]
Indeed. Thanks for reporting. Someone must have broken something in recent Glyphs updates.
This plugin and its code is incredibly old and utilised lots of hacks back then. When I find the time I will need to properly clean up.

Hope I can fix this very issue soon. Sorry for that!

**Comment by schriftgestalt** [comment]
Can you point me to the code where it should read the control letters?

**Comment by Mark2Mark** [comment]
about here:
https://github.com/bBoxType/Kernkraft/blob/cc6b987ec111222b6e7ce08204b5f2dafb76c3ef/Kernkraft.glyphsPlugin/Contents/Resources/KernKraftModule.py#L336

the customKenringStrings are defined in kernKit.py

```
"thai": {
	"Letter": {
		"Other": {
			"Left": "/thoThahan-thai/thoThahan-thai/doDek-thai/thoThahan-thai",
			"Right": "/thoThahan-thai/doDek-thai/doDek-thai/noNu-thai",
		},
		"Spacing": {
			"Left": "/khoKhwai-thai/khoKhwai-thai/doDek-thai/khoKhwai-thai",
			"Right": "/khoKhwai-thai/doDek-thai/doDek-thai/khoKhwai-thai",
		},
	},
},
```

**Comment by schriftgestalt** [comment]
`glyphs.subCategory` in not "Other" any more, it is just `None`.

**Comment by schriftgestalt** [comment]
I send a PR but didn’t test it.

**Comment by Mark2Mark** [comment]
Thanks! I will test later today ⏱️

**Comment by boomwooq** [comment]
@schriftgestalt Thank you! I did the test, works now
